doc/driver.mld

```diff
@@ -702,2157 +751,156 @@ html/odoc/odoc_html/Odoc_html/Html_fragment_json
-$ '../src/odoc/bin/main.exe' 'html-generate' 'tyxml_html.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'html_sigs.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'html_types.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'tyxml_svg.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'svg_sigs.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'svg_types.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'tyxml_xml.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'xml_stream.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'xml_sigs.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'xml_wrap.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' '--source' '../src/odoc/support_files.ml' 'odoc_odoc__Support_files.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' '--source' '../src/odoc/source_tree.ml' 'odoc_odoc__Source_tree.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' '--source' '../src/odoc/rendering.ml' 'odoc_odoc__Rendering.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' '--source' '../src/odoc/odoc_link.ml' 'odoc_odoc__Odoc_link.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' '--source' '../src/odoc/odoc_file.ml' 'odoc_odoc__Odoc_file.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' '--source' '../src/odoc/man_page.ml' 'odoc_odoc__Man_page.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' '--source' '../src/manpage/generator.ml' 'odoc_manpage__Generator.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' 'odoc_manpage.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' '--source' '../src/odoc/latex.ml' 'odoc_odoc__Latex.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' '--source' '../src/odoc/html_fragment.ml' 'odoc_odoc__Html_fragment.odocl' '-o' 'html' '--theme-uri' 'odoc' '--support-uri' 'odoc'
-$ '../src/odoc/bin/main.exe' 'html-generate' '--source' '../src/odoc/depends.ml' 'odoc_odoc__Depends.odo
```

Comment:
This introduces some irreproducibility in the doc generation, leading to an infinite loop of
```
$ dune build @docgen
$ dune promote
```
which never stabilizes...

Reply:
This comment should be located on the "show the k longest command to run"...